plugins/woocommerce/src/Admin/Features/LaunchYourStore.php

```diff
@@ -66,6 +67,30 @@ public function save_site_visibility_options() {
 		wc_admin_record_tracks_event( 'site_visibility_saved', $event_data );
 	}
 
+	/**
+	 * Append coming soon prop tracks globally.
+	 *
+	 * @param array $event_properties Event properties array.
+	 *
+	 * @return array
+	 */
+	public function append_coming_soon_global_tracks( $event_properties ) {
+		if ( is_array( $event_properties ) ) {
+			$coming_soon = 'no';
+			if ( 'yes' === get_option( 'woocommerce_coming_soon', 'no' ) ) {
+				$coming_soon = 'yes';
```

Comment:
I think you can remove this check. The options will always be "no"|"site"|"store" and never be "yes".